lua/cfc_notifications/shared/sh_presets.lua

```diff
@@ -64,20 +84,28 @@ CFCNotifications.registerNotificationType( "Buttons", function( CONTEXT )
             self:_addDefaultButtons()
         end
 
+        if self._curRowSize > 0 then
+            self._rowSizes = self._rowSizes or {}
+            table.insert( self._rowSizes, self._curRowSize )
+        end
+
         local w, h = canvas:GetSize()
+        local btnRow = 1
+        local btnCol = 1
         local btnGap = 20
-        local btnTotalW = ( w / #self._buttons )
+        local btnTotalW = ( w / self._rowSizes[btnRow] )
```

Comment:
Brackets aren't needed here
Noticed they were there before, so my fault, but still should be removed :)

Reply:
They are needed, I'm looking for the index equal to the value of btnRow, not an index called btnRow.